django/core/signing.py

```diff
@@ -90,10 +101,17 @@ def loads(self, data):
         return json.loads(data.decode('latin-1'))
 
 
-def dumps(obj, key=None, salt='django.core.signing', serializer=JSONSerializer, compress=False):
+def dumps(
+    obj,
+    key=None,
+    old_keys=None,
```

Comment:
the old keys aren't needed when dumping so I don't think this function needs to change